packages/react-components/lib/dispensers/dispenser-accordion.tsx

```diff
@@ -1,20 +1,21 @@
-import { Accordion, AccordionProps, makeStyles, Divider } from '@material-ui/core';
-import Typography from '@material-ui/core/Typography';
+import { Accordion, AccordionProps, makeStyles } from '@material-ui/core';
 import * as RomiCore from '@osrf/romi-js-core-interfaces';
 import Debug from 'debug';
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
```

Comment:
stick to a convention whether to use default import for "destructured" import, in general we are using default import for react. I don't really mind if you prefer "destructured", if you would like to use it, change the other references to remove `React.` as well.